fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -374,12 +373,11 @@
 			/// Otherwise, check if it's Sol, to enter Torpor.
 			if(clan.bloodsucker_sunlight.amDay)
 				Check_Begin_Torpor(TRUE)
-		/// You are in Torpor, and in a Coffin. Check if it's not Daytime & you have less than 10 Brute/Burn combined to end Torpor.
-		else if(!clan.bloodsucker_sunlight.amDay && total_damage <= 10)
+		/// You are in Torpor, and in a Coffin. Check if you are supposed to end torpor
+		else
 			Check_End_Torpor()
```

Comment:
I don't want Check_End_Torpor being called every single lifetick if its not needed.

Reply:
It'll be about the same either way, since Check_End_Torpor performs all the same checks that were being performed to determine whether or not one should call the proc to begin with. Check_End_Torpor was always called when it would guarantee that it would pass, which kind of de-necessitated Check_End_Torpor